src/Compilers/Core/Portable/Syntax/LineDirectiveMap.cs

```diff
@@ -19,7 +20,7 @@ namespace Microsoft.CodeAnalysis
     internal abstract partial class LineDirectiveMap<TDirective>
         where TDirective : SyntaxNode
     {
-        protected readonly LineMappingEntry[] Entries;
+        protected readonly ImmutableArray<LineMappingEntry> Entries;
```

Comment:
@AlekseyTs do you mean the change to this file or the entire PR? The change to this file just changes the types from array to ImmutableArray, and makes the fetching of source text more lazy.